uploaders/apkuploader_test.go

```diff
@@ -0,0 +1,371 @@
+package uploaders
+
+import "testing"
+
+func Test_filterPackageInfos(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		aaptOut string
+		want    string
+		want1   string
+		want2   string
+	}{
+		{
+			name: "test_with_empty_platformBuildVersionName",
+			aaptOut: `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0' platformBuildVersionName=''
+		sdkVersion:'17'
```

Comment:
please do not use the whole output for every test.
It is fine to test the whole output once, but other than this, just test the relevant line of the log (or some log from its environment).
Also in case of the whole output test, please define the whole output using a var, to make the tests more readable.